apps/src/codebridge/codebridgeContext/codebridgeContextProjectReducer.ts

```diff
@@ -180,11 +185,19 @@ export const projectReducer = (project: ProjectType, action: ReducerAction) => {
     case PROJECT_REDUCER_ACTIONS.DELETE_FILE: {
       const {fileId} = <DefaultFilePayload>action.payload;
 
+      const openFileIds = getOpenFileIds(project);
+      const newOpenFileIds = openFileIds.find(
+        openFileId => openFileId === fileId
+      )
+        ? openFileIds.filter(openFileId => openFileId !== fileId)
+        : openFileIds;
+
```

Comment:
This was a subtle data duplication bug where we'd create a new `openFileIds` array even if we don't need one.